crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -226,11 +233,11 @@ def save_and_restore
       return false unless @backup.save
       if Crowbar::Backup::Restore.restore_steps_path.exist?
         flash[:info] = t("installer.upgrades.restore.multiple_restore")
-        true
       else
         Crowbar::Backup::Restore.purge
         @backup.restore(background: true, from_upgrade: true)
       end
+      true
```

Comment:
So maybe that's what we should fix? I mean, if it returns `false` in some case, then it sounds wrong to just always return `true` here.